mapeditor/mapsettings/eventsettings.cpp

```diff
@@ -41,8 +41,8 @@ QVariant toVariant(const CMapEvent & event)
 	result["players"] = QVariant::fromValue(event.players);
 	result["humanAffected"] = QVariant::fromValue(event.humanAffected);
 	result["computerAffected"] = QVariant::fromValue(event.computerAffected);
-	result["firstOccurence"] = QVariant::fromValue(event.firstOccurence);
-	result["nextOccurence"] = QVariant::fromValue(event.nextOccurence);
+	result["firstOccurrence"] = QVariant::fromValue(event.firstOccurrence);
```

Comment:
does this have chance to break compatibility with existing maps?

Reply:
Somewhat. Maps would still load, but events may not be configured correctly (assuming that we have maps in vcmi format with daily events)